addons/sms/wizard/sms_composer.py

```diff
@@ -71,7 +71,7 @@ def default_get(self, fields):
     template_id = fields.Many2one('sms.template', string='Use Template', domain="[('model', '=', res_model)]")
     body = fields.Text(
         'Message', compute='_compute_body',
-        precompute=True, readonly=False, store=True, required=True)
+        precompute=True, readonly=False, store=True)
```

Comment:
Why this change ? I don't think we want to send void SMS ...

Reply:
The required is now conditional in the view: 
https://github.com/odoo/odoo/pull/83745/files#diff-dd4eeb56d7e858242e106057240dfb4ac532476372c8188f793a90ad3364bfd6R50-R51
The issue was that the field being required in the `wizard/sms_composer.py` file and the body field appearing twice in the view: 
https://github.com/odoo/odoo/pull/83745/files#diff-dd4eeb56d7e858242e106057240dfb4ac532476372c8188f793a90ad3364bfd6L51-L52
make two invalid fields error for the same field if left empty and trying to send a SMS https://imgur.com/AAX0wIq

Reply:
Well we cannot remove the required at model level ... sending a void email is ok as you may have some other information, not sending a void sms ^^ . It that is the only condition I would say tant pis and keep the required on the field itself.

Reply:
Got it, thanks! The field is required=True again